Java/src/test/java/AirportTest.java

```diff
@@ -65,7 +61,7 @@ public void test3() {
         for (int i = 0; i < planesSortedByMaxLoadCapacity.size() - 1; i++) {
             Plane currentPlane = planesSortedByMaxLoadCapacity.get(i);
             Plane nextPlane = planesSortedByMaxLoadCapacity.get(i + 1);
-            if (currentPlane.getMinLoadCapacity() > nextPlane.getMinLoadCapacity()) {
+            if (currentPlane.getMaxLoadCapacity() > nextPlane.getMaxLoadCapacity()) {
                 nextPlaneMaxLoadCapacityIsHigherThanCurrent = false;
                 break;
             }
```

Comment:
this is again not a good well-described check. instead I would better compare 2 lists - 1 originally sorted by the method under the test (`.sortByMaxLoadCapacity`), and second - the copy of original list sorted using java collections API